microsoft-edge/devtools-protocol/index.md

```diff
@@ -0,0 +1,53 @@
+# Microsoft Edge DevTools Protocol
+
+Developer tools can use the **Microsoft Edge DevTools Protocol** to inspect and debug the Microsoft Edge browser and other hosts of the EdgeHTML, such as Progressive Web Apps (PWAs that run in the *WWAHost.exe* process) and the Universal Windows Platform webview control. It provides a set of methods and events that are organized into different [Domains]() of EdgeHTML engine instrumentation.
+
+ You can call these methods and monitor these events through raw JSON messages passed across a web socket connection between a remote (debuggee) and host (debugger) process. Microsoft Edge DevTools uses this protocol to enable [remote debugging](0.1/clients.md#microsoft-edge-devtools) of a host machine running Microsoft Edge from the standalone DevTools client available from the [Microsoft Store]().
+
+The Microsoft Edge DevTools Protocol is designed to be interoperable with the [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/). 
+
+## Using the protocol
+
+Here's how to attach a custom tooling client to a running instance of Microsoft Edge. See the [remote debugging](0.1/clients.md#microsoft-edge-devtools) instructions if you're using Microsoft Edge DevTools as your client.
+
+1. Launch Microsoft Edge with the remote debugging port open, specifying the URL you wish to open. For example:
+
+    ```
+    MicrosoftEdge.exe --devtools-server-port 9222 https://www.bing.com
+    ```
+
+    If Edge is already launched, the URL parameter is optional. A  button will appear next to the browser address bar to indicate  the **Developer tools server** has started:
+
+    ![Developer tools server](media/developer-tools-server.png) 
+
+2. Use this [HTTP endpoint](0.1/http.md) to get a list of attachable page targets:
+
+    ```
+    http://localhost:9222/json/list
+    ```
+
+3. Connect to the listed `webSocketDebuggerUrl` of the desired page to issue further [protocol commands](0.1/index.md) and receive event messages through the devtools socket server.
+
+## Status and feedback
+
+The preliminary release ([Version 0.1]()) of the Microsoft Edge DevTools Protocol provides core debugging functionality, such as setting breakpoints, stepping through code, and exploring stack traces. In the Edge DevTools UI, this translates to functionality available in the [**Debugger**](../devtools-guide/debugger.md) panel, minus cache inspection (for Web storage, Service worker, Cache API, and IndexedDB).
+
+Further platform instrumentation will be available in future releases of the protocol.
+
+Thanks for trying the Edge DevTools Protocol! We'd love to hear your feedback at:
+
+ - [**Microsoft Edge Developer UserVoice**](https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer?category_id=84475): DevTools feature ideas and requests
+
+ - [**EdgeHTML Issue Tracker**](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/): Protocol, DevTools, and EdgeHTML platform bugs and issues
+
```

Comment:
Feedback can also be submitted via the feedback hub, which would be good to drive people to, we can get extra data from the machines when there are actual crashes or bugs.

Reply:
Link building for Feedback Hub can be found [here](https://osgwiki.com/wiki/Feedback_-_Launching_Feedback_Hub_using_URI). I'll create one and follow up here.

Reply:
link to feedback hub for [DevTools](Feedback-Hub:?referrer=microsoftEdge&tabID=2&newFeedback=true&ContextId=344)